client/ayon_openrv/plugins/load/load_mov.py

```diff
@@ -15,9 +15,9 @@ class MovLoader(load.LoaderPlugin):
     """Load mov into OpenRV"""
 
     label = "Load MOV"
-    families = ["*"]
-    representations = ["*"]
-    extensions = ["mov", "mp4"]
+    product_types = {"*"}
+    representations = {"*"}
```

Comment:
This is still a `list` in ayon core currently: https://github.com/ynput/ayon-core/blob/b38f7132ca0c162f6d99274f3b6701b29723c632/client/ayon_core/pipeline/load/plugins.py#L27

```suggestion
    representations = ["*"]
```